client/lib/user/support-user-interop.js

```diff
@@ -125,7 +125,8 @@ export const supportUserBoot = () => {
 	onBeforeUnload = storeUserAndToken( user, token );
 	window.addEventListener( 'beforeunload', onBeforeUnload );
 
-	localforage.bypass();
+	bypassPersistentStorage( true );
+	clearStorage();
```

Comment:
should we just bake this call into enabling bypass storage?

Reply:
That's one possibility. I opted for "consumers choose the semantics", but we could just as well enforce it. There's probably not a lot of use-cases for reusing what was already in memory, right? It's supposed to be volatile anyway.

Reply:
yeah, and it would let us hide the sync nature of the bypass clear inside the enabler. This call feels weird to me right now because it returns a Promise that we don't listen to.

Reply:
Done. This made `bypassPersistentStorage` an async method too, so I updated everything accordingly.